src/js/components/Navigation/HeaderBar.jsx

```diff
@@ -354,10 +354,11 @@ class HeaderBar extends Component {
 
   onAnalyticsStoreChange () {
     // A reload after facebook login forces the need for a voterRetrieve, after redrawing the page
-    // Without making changes to the API server, the first response that indicates 'is signed in' is an Analytics call response
-    if (VoterStore.getVoterIsSignedIn() !== true && (AnalyticsStore.getIsSignedIn() || FacebookStore.loggedIn)) {
+    // (and without requiring changes to the API server), the first response that indicates 'is signed in' is an Analytics call response
+    // console.log('onAnalyticsStoreChange VoterStore.getVoterIsSignedIn(): ', VoterStore.getVoterIsSignedIn(), ' AnalyticsStore.getIsSignedIn(): ', AnalyticsStore.getIsSignedIn(), 'FacebookStore.loggedIn: ', FacebookStore.loggedIn, 'VoterStore.voterDeviceId(): ', VoterStore.voterDeviceId());
+    if (isCordova() && VoterStore.getVoterIsSignedIn() === false && (AnalyticsStore.getIsSignedIn() || FacebookStore.loggedIn)) {
```

Comment:
Does this need to be limited to "isCordova"? I worry this might be needed in the browser as well.

Reply:
This is a side effect of a ugly fix i made for being zoomed in in iOS after signing into facebook.  
In the next PR the comment says
    // A page reload for iOS in Cordova after facebook login forces the need for a voterRetrieve, after redrawing the page

This is only for Cordova.  The page reload basically reboots the app.